defaults/main.yml

```diff
@@ -27,3 +27,7 @@ rstudio_connect_config:
     URL: https://packagemanager.rstudio.com/cran/__linux__/{{ ansible_lsb.codename | lower }}/latest
   'RPackageRepository "RSPM"':
     URL: https://packagemanager.rstudio.com/cran/__linux__/{{ ansible_lsb.codename | lower }}/latest
+
+python_versions: []
+rstudio_connect_python_executables: []
+rstudio_connect_enable_python: "{{ true if (python_versions | length or rstudio_connect_python_executables | length) else false }}"
```

Comment:
I'm not 100% sure if it's good idea to use such conditional here (but it works - if `python_versions` or `rstudio_connect_python_executables` are specified by user, it will set `rstudio_connect_enable_python` as true)

Reply:
If it works as expected then it's fine for me!

Reply:
`rstudio_connect_enable_python` was removed and the conditional was moved to [the template](https://github.com/Appsilon/ansible-rstudio-connect/pull/24/files#diff-dd5e9393b71623e167fcfaeb7055d6c619a68a7ca3194e18b7a72de364e5628dR11)